pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go

```diff
@@ -48,6 +48,10 @@ func (defaultauthZ *DefaultAuthZImpl) Handle(command utils.CommandEnum, cluster
 			if err := NetworkAuthorization(cluster, r, containerConfig.HostConfig.NetworkMode); err != nil {
 				return err
 			}
+			if err := hostFSMountCheck(r.Header.Get(headers.AuthZTenantIdHeaderName), containerConfig.HostConfig.Binds); err != nil {
+				return err
+			}
+
```

Comment:
> From: Ezra Silvera <notifications@github.com>
> To: swarm-hooks/swarm <swarm@noreply.github.com>
> Cc: Kenneth Nagin/Haifa/IBM@IBMIL, Author <author@noreply.github.com>
> Date: 19/07/2016 04:24 PM
> Subject: Re: [swarm-hooks/swarm] Volume binding (#46)
> 
> In pkg/multiTenancyPlugins/authorization/defaultAuthZImpl.go:
> > @@ -48,6 +48,10 @@ func (defaultauthZ *DefaultAuthZImpl) Handle
> (command utils.CommandEnum, cluster
> >           if err := NetworkAuthorization(cluster, r, 
> containerConfig.HostConfig.NetworkMode); err != nil {
> >              return err
> >           }
> > +         if err := hostFSMountCheck(r.Header.Get
> (headers.AuthZTenantIdHeaderName), 
> containerConfig.HostConfig.Binds); err != nil {
> > +            return err
> > +         }
> > +

> Should this be moved to the argument enforcement ? 
> ?
Indeed, if we had some other "argument enforcement" code this would be a 
candidate.
However, no such code exists nor do we have a design whether we want it 
fit the model
of api filtering.  Once we understand how "argument enforcement" will be 
done
then we can decide whether to include -v argument handling.
For now I think it is best to include it in the authorization plugin.